templates/csi-clusterrole.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and (eq (.Values.csi.enabled | toString) "true" ) (eq (.Values.global.enabled | toString) "true") }}
+{{- if eq (include "vault.csiEnabled" . | toString) "true" -}}
```

Comment:
Sadly the template `define` implicitly converts booleans to strings, so we can't just do `if (include "vault.csiEnabled" .)`